libraries/cyclestreets-view/src/main/res/layout/activity_circular_route.xml

```diff
@@ -23,11 +25,14 @@
         android:layout_below="@id/textView2"
         android:layout_marginTop="16dp"
 
+        android:paddingStart="8dp"
         android:paddingTop="8dp"
+        android:paddingEnd="8dp"
         android:paddingBottom="8dp"
         app:tabGravity="fill"
         app:tabIndicatorColor="@color/apptheme_color"
         app:tabIndicatorHeight="4dp"
+        app:tabMaxWidth="0dp"
```

Comment:
In Landscape mode the tabs were squashed into the middle of the screen despite having "match_parent on the width". (Sorry I don't have a screenshot of that.)
tabMaxWidth="0dp" allows the tabs to expand to full screen width.